yt_dlp/extractor/googledrive.py

```diff
@@ -129,7 +129,7 @@ def _get_captions_by_type(self, video_id, subtitles_id, caption_type,
                     'lang': (caption_lang_code if origin_lang_code is None
                              else origin_lang_code),
                     'type': 'track',
-                    'name': '',
+                    'name': caption_entry.attrib.get('name', ''),
```

Comment:
```suggestion
                    'name': caption_entry.attrib.get('name') or '',
```
or if we can't get subs at all without this value, should we `report_warning` and `continue` above?